.github/actions/install_with_cache/action.yml

```diff
@@ -10,7 +10,7 @@ runs:
         path: |
           node_modules
           packages/**/node_modules
-        key: ${{ runner.os }}-${{ hashFiles('package-lock.json') }}
+        key: ${{ runner.os }}-${{ hashFiles('package-lock.json') }}-v2
```

Comment:
This is needed to invalidate cache that's incompatible with larger worker and is leading to:
```
Prepare all required actions
Getting action download info
Download action repository 'actions/cache@885[2](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:2)2ab9f39a2ea568f7027eddc7d8d8bc9d59c8' (SHA:88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8)
Run ./.github/actions/restore_build_cache
Prepare all required actions
Getting action download info
Run ./.github/actions/restore_install_cache
Run actions/cache@88522ab9f[3](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:3)9a2ea568f7027eddc7d8d8bc9d59c8
Received 13[4](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:4)217728 of 24[5](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:6)825942 (54.[6](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:7)%), 126.2 MBs/sec
Received 245825942 of 245825942 (100.0%), 133.0 MBs/sec
Cache Size: ~234 MB (245825942 B)
"C:\Program Files\Git\usr\bin\tar.exe" -xf C:/a/_temp/00c0e1[7](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:8)2-e349-4f14-92f9-10005a622fb[8](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:10)/cache.tzst -P -C C:/a/amplify-backend/amplify-backend --force-local --use-compress-program "zstd -d"
Cache restored successfully
Cache restored from key: Windows-418f36fb2696ff545b34d4f90f527534293b1ff5110a4729ea71303b9b53485d
Run actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8
Received 1342[17](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:20)728 of 167938[22](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:25)8 (79.9%), 114.7 MBs/sec
Received 167938228 of 167938228 (100.0%), 116.9 MBs/sec
Cache Size: ~160 MB (167938228 B)
"C:\Program Files\Git\usr\bin\tar.exe" -xf C:/a/_temp/bb0ebe96-47ed-43e9-890f-ead3ad[23](https://github.com/aws-amplify/amplify-backend/actions/runs/6899407583/job/18770983822#step:4:26)ea8b/cache.tzst -P -C C:/a/amplify-backend/amplify-backend --force-local --use-compress-program "zstd -d"
/usr/bin/tar: node_modules/@aws-amplify/auth-construct-alpha/lib: Cannot mkdir: No medium found
/usr/bin/tar: node_modules/@aws-amplify/auth-construct-alpha/lib/defaults.d.ts.map: Cannot open: No medium found
/usr/bin/tar: node_modules/@aws-amplify/auth-construct-alpha/lib/string_maps.d.ts.map: Cannot open: No medium found
/usr/bin/tar: node_modules/@aws-amplify/auth-construct-alpha/lib/construct.js: Cannot open: No medium found
/usr/bin/tar: node_modules/@aws-amplify/auth-construct-alpha/lib/string_maps.js: Cannot open: No medium found
/usr/bin/tar: node_modules/@aws-amplify/auth-construct-alpha/lib/trigger_events.d.ts: Cannot open: No medium found
/usr/bin/tar: node_modules/@aws-amplify/auth-construct-alpha/lib/index.js: Cannot open: No medium found
/usr/bin/tar: node_modules/@aws-amplify/auth-construct-alpha/lib/trigger_events.js: Cannot open: No medium found
```